stl/inc/vector

```diff
@@ -2882,6 +2882,62 @@ namespace pmr {
     using vector = _STD vector<_Ty, polymorphic_allocator<_Ty>>;
 } // namespace pmr
 #endif // _HAS_CXX17
+
+#if _HAS_IF_CONSTEXPR
+// VARIABLE TEMPLATE _Is_vb_iterator
+template <class _Alloc>
+_INLINE_VAR constexpr bool _Is_vb_iterator<_Vb_const_iterator<_Alloc>> = true;
+
+template <class _Alloc>
+_INLINE_VAR constexpr bool _Is_vb_iterator<_Vb_iterator<_Alloc>> = true;
+
+template <class _FwdIt, class _Ty>
+_CONSTEXPR20 void _Fill_vbool(_FwdIt _First, _FwdIt _Last, const _Ty& _Val) {
+    // Set [_First, _Last) to _Val
+    if (_First == _Last) {
+        return;
+    }
+
+    _Vbase* _UFirst            = const_cast<_Vbase*>(_First._Myptr);

```

Comment:
Hmmm not a fan of using `_UFirst` to mean this -- that convention is such that it's the unchecked form of `_First` but here it's completely unrelated to `_First`.

Reply:
I mean in the ranges world it counts as an unwrapped iterator which it kind of is.

I am happy to take any better naming you propose

Reply:
>I mean in the ranges world it counts as an unwrapped iterator which it kind of is.

Not really, unwrapped means that it has the debugging support etc. stripped off.

>I am happy to take any better naming you propose

`_First_vbase` ? `_First_vb`? `_Vb_first`? Anything to indicate that these are the unsigned int domain rather than _First's bit domain.